client/client/modules/render/tracks/reference/menu/generalMenu.js

```diff
@@ -1,6 +1,15 @@
+import {menu} from '../../../utilities';
 import downloadFile from '../../common/menu/downloadFile';
 import header from '../../common/menu/header';
-import {menu} from '../../../utilities';
+
+function openLineage(tracks) {
+    const [dispatcher] = (tracks || [])
+        .map(track => track.config.dispatcher)
+        .filter(Boolean);
+    if (dispatcher) {
+        dispatcher.emitSimpleEvent('read:show:lineage');
```

Comment:
Change event name to `reference:show:lineage` (as it has no relations to the *reads* of BAM track)